src/web/themes/custom/bcgov_career/bcgov_career.theme

```diff
@@ -45,6 +45,7 @@ function bcgov_career_preprocess_node(&$variables) {
     $variables['quiz_color'] = _quiz_color_type($quiz_type);
     $variables['test_career'] = 'test';
     $variables['attributes']['class'][] = $variables['quiz_color'];
+    $variables['workbc_url'] = getenv("WORKBC_URL");
```

Comment:
Please do the `rtrim('/') . '/'` trick to make sure we don't fail because of a backslash.

Reply:
fixed